### DISCUSSION TITLE
根据Loss值图选择较佳模型的讨论

### DISCUSSION THREAD

**Discussion by JerryKingQAQ** [svc-develop-team/so-vits-svc#321] [General]
![image](https://github.com/svc-develop-team/so-vits-svc/assets/95563982/71ee4b6e-65d2-4154-9b2f-b42d08a4dd4a)

speaker人数为2，一位3小时语音加唱歌纯人声素材，一位2小时语音加唱歌纯人声素材。

请问如何根据Loss图找到较佳的模型？

**Comment by JerryKingQAQ** [comment]
![image](https://github.com/svc-develop-team/so-vits-svc/assets/95563982/ae868bb1-fb5e-4e81-bacb-9783a34a1fb2)
再补一张图，配合opencpop/m4singer里的所有数据进行训练

总共23个speaker，其中2个speaker是我训练目标，剩余21个speaker为opencpop/m4singer里的数据

**Comment by bfloat16** [comment]
理论上loss是低点比较好
我的方法是直接推理测试：
将推理源放入dataset_raw文件夹内跟随训练数据一起预处理，在预处理完毕后将推理源的路径从从train.txt剪切到val.txt，训练的时候直接听gen的效果就可以了

**Reply by iiallgaii** [reply]
interesting, may i know can u make a demo video to us? thanks
